pandas/core/arrays/base.py

```diff
@@ -49,6 +51,9 @@
 from pandas.core.missing import get_fill_func
 from pandas.core.sorting import nargminmax, nargsort
 
+if TYPE_CHECKING:
+    from typing import Literal
```

Comment:
Why only import this one if TYPE_CHECKING? (asking because I don't know, not because I think it's wrong)

Reply:
`Literal` was only used in one other place `core/frame.py`, so I followed the pattern there.  I think the issue is that `Literal` was introduced in python 3.8, so doing it with if `TYPE_CHECKING` makes sure that it is compatible with python 3.7 .